modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5511,6 +5511,7 @@ deleting-this-event-will-cancel-the-meeting-with-your-guests-would-you-like-to-d
 deletion-conflicts-with-a-modification=You are attempting to delete an entity that was recently updated in production. Please discard the deletion. Then, publish your remaining changes or delete the entity again before proceeding.
 deletion-modification-conflict=Missing Entity
 deletion-not-allowed=Deletion Not Allowed
+deletion-not-allowed-because-a-relationship-field-is-being-used-in-the-object-definition-x-to-proceed-introduce-a-new-custom-field-in-x=Deletion not allowed because a relationship field is being used in the object definition "{0}". To proceed, introduce a new custom field in "{1}".
```

Comment:
@larissacribeiro When we have more than one sentence in the language key value, you can define the key until the end of the first sentence, like: `deletion-not-allowed-because-a-relationship-field-is-being-used-in-the-object-definition-x=Deletion not allowed because a relationship field is being used in the object definition "{0}". To proceed, introduce a new custom field in "{1}".`

Reply:
@naraandrade that's nice to know, thanks!